packages/web/src/react.test.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable testing-library/no-node-access */
```

Comment:
What's the reason for not using the `testing-library` helpers, and instead accessing the dom ourselves?

Reply:
Just tried it if `testid` – unfortunately without luck